src/containers/App/App.js

```diff
@@ -35,6 +35,9 @@ export class App extends Component {
         <header>
           <h1>Movie Tracker</h1>
           <div>
+            {
+              this.props.user.name ? <span className="user-name">Welcome: {this.props.user.name}... </span> : ''
+            }
```

Comment:
This is a great feature, maybe we can style it with a h3/h4 tag to display above the nav bar